build/cve-lists/cve-list-product-1.3.0.txt

```diff
@@ -0,0 +1,6 @@
+CVE-2024-24790
```

Comment:
would it make more sense to locate these files in the modules/release-notes/ folder?